model/project_ref.go

```diff
@@ -1078,6 +1078,21 @@ func FindFirstProjectRef() (*ProjectRef, error) {
 	return projectRef, nil
 }
 
+// FindAllMergedTrackedEnabledProjectRefs returns all enabled project refs in the db
+// that are currently being tracked (i.e. their project files
+// still exist and the project is not hidden).
+// Can't hide a repo without hiding the branches, so don't need to aggregate here.
```

Comment:
This isn't super relevant to this function given my comment below, i.e. that we _do_ need to consider the repo value.